src/Dropdown/index.js

```diff
@@ -121,7 +122,7 @@ Dropdown.propTypes = {
   /** Initial label shown in input */
   triggerLabel: PropTypes.string.isRequired,
   /** Array of elements to render as items in the dropdown */
-  children: PropTypes.arrayOf(PropTypes.element),
```

Comment:
noticed this error 
checkPropTypes.js:20 Warning: Failed prop type: Invalid prop `children[0]` of type `array` supplied to `Dropdown`, expected a single ReactElement